scripts/generate/templates/PRODUCT_SUBSCRIPTION/react.template.js

```diff
@@ -129,15 +129,17 @@ function Create() {
     done();
   }, [getSessionToken, done]);
 
-  const actions = useMemo(
-    () => (
-      <Stack spacing="none" distribution="fill">
-        <Button title="Cancel" onPress={() => close()} />
-        <Stack distribution="trailing">
-          <Button title="Create plan" onPress={onPrimaryAction} primary />
-        </Stack>
+  const Actions = ({onPrimary, onClose}) => (
```

Comment:
I think this should be a functional component separate from this one - otherwise we are still not following conventions

Reply:
I'm not sure I understand what you mean here. `Actions` is functional component. How would you separate it further?

Reply:
I mean it should be its own component instead of inside `Create`. The current approach creates an inline object called Actions every render which is then memorized to `cachedActions`

Reply:
Got it. I actually wanted to refactor this so that we could reuse the `Actions` component anyway. If I pull it out to the top level, that is possible. 😃